internal/server/server.go

```diff
@@ -94,9 +95,9 @@ func createSOMASServer(
 				shared.Judge:     0,
 				shared.Speaker:   0,
 			},
-			SpeakerID:   shared.Team1,
-			JudgeID:     shared.Team2,
-			PresidentID: shared.Team3,
+			SpeakerID:   shared.ClientID(rand.Intn(len(shared.TeamIDs))),
+			JudgeID:     shared.ClientID(rand.Intn(len(shared.TeamIDs))),
+			PresidentID: shared.ClientID(rand.Intn(len(shared.TeamIDs))),
```

Comment:
there is a PR from Toby coming assigning 3 different random islands, just fyi

Reply:
see #371 :D

Reply:
Yeah I had a look at #371, why is it so long, doesn't this method do the same thing in 3 lines?

Reply:
I spoke to Rudolfs about it and we wanted the three roles to be unique at the start.

Reply:
I need to speak with @Spruudis

Reply:
like, I don't remember him saying this to me

Reply:
@Spruudis  and WHOMEVER needs to be, 

quick call in IIGO server?

Reply:
Ok fixed